hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -451,7 +451,7 @@ private static int doBootstrap(JavaSparkContext jsc, String tableName, String ta
   }
 
   private static int rollback(JavaSparkContext jsc, String instantTime, String basePath, Boolean rollbackUsingMarkers) throws Exception {
-    SparkRDDWriteClient client = createHoodieClient(jsc, basePath, rollbackUsingMarkers);
+    SparkRDDWriteClient client = createHoodieClient(jsc, basePath, rollbackUsingMarkers, false);
```

Comment:
This call here is supposed to call method at L529 [here](https://github.com/apache/hudi/blob/cd314b74eabaf187ce073ec938a6a38e502288f1/hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java#L529), but currently it's calling method at L534 [here](https://github.com/apache/hudi/blob/cd314b74eabaf187ce073ec938a6a38e502288f1/hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java#L534), and it's passing `rollbackUsingMarkers` as `lazyCleanPolicy`, which would cause issues when `rollbackUsingMarkers` is passed in hudi-cli

Reply:
Is there any possibility we write a test here ?

Reply:
We can but adding a test is non-trival work, as it involves another option introduced in this [PR](https://github.com/apache/hudi/pull/5158/files)
We might want to file a seperate jira to add a test.

Apart from that, I tested this fix manually on AWS EMR

Reply:
Let's try to see if we can add a test please. would be helpful in general.

Reply:
I've added a test for this change. Could you take a look again? Also I've noticed a flink IT test has been failing: https://dev.azure.com/apache-hudi-ci-org/apache-hudi-ci/_build/results?buildId=9375&view=logs&j=3b6e910d-b98f-5de6-b9cb-1e5ff571f5de&t=30b5aae4-0ea0-5566-42d0-febf71a7061a&l=120490

I don't think my change would affect this test, but not sure if this is a flaky test, could you also take a look?